### PR TITLE
Admin API to refresh all tasks of a workflow

### PR DESCRIPTION
**Pull request by anish531213**

### CODE REVIEW

File: thrift/history.thrift [hunk 1]
```diff
@@ -857,4 +862,19 @@ service HistoryService {
...
+  /**
+  * RefreshWorkflowTasks refreshes all tasks of a workflow
+  **/
+  void RefreshWorkflowTasks(1: RefreshWorkflowTasksRequest request)
+    throws (
+      1: shared.BadRequestError badRequestError,
+      2: shared.InternalServiceError internalServiceError,
+      3: shared.EntityNotExistsError entityNotExistError,
+      4: ShardOwnershipLostError shardOwnershipLostError,
+      5: shared.LimitExceededError limitExceededError,
```
[the remark this answers is not in the document]
Comment (anish531213) [reply thrift/history.thrift]: Yeah...that's true. I will update that.

File: thrift/history.thrift [hunk 2]
```diff
@@ -871,10 +871,8 @@ service HistoryService {
     throws (
       1: shared.BadRequestError badRequestError,
       2: shared.InternalServiceError internalServiceError,
-      3: shared.EntityNotExistsError entityNotExistError,
```
Comment (yycptt) [inline thrift/history.thrift]: I think this error still applies. If the API is called on a workflow that doesn't exist, this error should be returned. It's different from the no-op case when the workflow has already closed.

File: thrift/admin.thrift [hunk 3]
```diff
@@ -141,6 +141,16 @@ service AdminService {
...
     )
+
+  /**
+  * RefreshWorkflowTasks refreshes all tasks of a workflow
+  **/
+  void RefreshWorkflowTasks(1: shared.RefreshWorkflowTasksRequest request)
+   throws (
+      1: shared.BadRequestError badRequestError,
+      2: shared.DomainNotActiveError domainNotActiveError,
+      3: shared.ServiceBusyError serviceBusyError,
```
Comment (yycptt) [inline thrift/admin.thrift]: Please also add the `entityNotExistError` here.

Comment (anish531213) [reply thrift/admin.thrift]: Done

File: thrift/admin.thrift [hunk 4]
```diff
@@ -141,6 +141,17 @@ service AdminService {
...
       2: shared.ServiceBusyError serviceBusyError,
     )
+
+  /**
+  * RefreshWorkflowTasks refreshes all tasks of a workflow
+  **/
+  void RefreshWorkflowTasks(1: shared.RefreshWorkflowTasksRequest request)
+   throws (
+      1: shared.BadRequestError badRequestError,
+      2: shared.DomainNotActiveError domainNotActiveError,
```
Comment (yycptt) [inline thrift/admin.thrift]: If this error is returned by the history client, also add it to `history.thrift`

**Pull request merged**